lib/hawk/model/association.rb

```diff
@@ -73,17 +73,16 @@ module ClassMethods
         def inherited(subclass)
           super
 
-          parent = self
           subclass.instance_eval do
             # Inherit associations
             @_associations ||= {}
 
-            parent.associations.each do |name, (type, options)|
+            self.associations.each do |name, (type, options)|
```

Comment:
-----> you would then try copying the associations from yourself, and end up with an empty associations list :smile: